device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/IotHubTransportConnection.java

```diff
@@ -60,7 +60,7 @@
      * @return true if the acknowledgement was sent successfully, and false otherwise
      * @throws TransportException if an exception occurred while sending the acknowledgement
      */
-    boolean sendMessageResult(Message message, IotHubMessageResult result) throws TransportException;
```

Comment:
Every call to this API was already passing in an IotHubTransportMessage type (which is a derived type from Message), and it made the amqp refactor easier for this interface to just take a transport message (which has callback info that Message does not)

Reply:
Is this change contract breaking?

Reply:
No, this is an internal interface